.woodpecker/test.yml

```diff
@@ -7,20 +7,27 @@ when:
     branch: "renovate/*"
 
 steps:
+  vendor:
+    image: *golang
+    commands:
+      - go mod vendor
+
   lint:
     image: *golang
+    depends_on: [ vendor ]
     commands:
       - make vet
       - make formatcheck
 
   test:
     image: *golang
+    depends_on: [ vendor ]
     commands:
       - make test
 
   build:
     image: woodpeckerci/plugin-docker-buildx:4.0.0
-    depends_on: [ lint, test ]
+    depends_on: [ lint, test, vendor ]
```

Comment:
This does not seem needed as lint and test already depend no vendor

Reply:
yes but if in future we alter stuff ... that step do explizit need vendor too ... here it is a noop but the documentation porpuse stayews